src/Popover.js

```diff
@@ -0,0 +1,234 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import classnames from 'classnames'
+import {get} from './constants'
+import theme from './theme'
+import elementType from './utils/elementType'
+import Box from './Box'
+import BorderBox from './BorderBox'
+
+const Popover = styled(Box).attrs(({className, caret}) => {
+  return {
+    className: classnames(className, `caret-pos--${caret}`)
+  }
+})`
+  position: ${props => (props.relative ? 'relative' : 'absolute')};
+  z-index: 100;
+  display: ${props => (props.open ? 'block' : 'none')};
+`
+
+Popover.Content = styled(BorderBox)`
```

Comment:
Same comment as the one above - if you want to allow users to override any of the styles you've authored here (margin, padding, background color, width) that correspond to `BorderBox` or `COMMON` props you'd need to add ${BORDER} to the end of this list of styles.